boards/budgy/main.py

```diff
@@ -26,22 +29,43 @@
     data_pin2=data_pin2,
 )
 
-layers = Layers()
 holdtap = HoldTap()
 mediaKeys = MediaKeys()
 combos = Combos()
+tapdance = TapDance()
 
-keyboard.modules = [layers, split, holdtap, mediaKeys, combos]
+mousekeys = MouseKeys(
+    max_speed=25,
+    acc_interval=17,  # Delta ms to apply acceleration
+    move_step=1
+)
+led = digitalio.DigitalInOut(board.LED)
+led.direction = digitalio.Direction.OUTPUT
 
 
-# Todo: Import either keymap_sw (for Swedish Colemak-DH) or keymap_us (for English qwerty)
+# Enable LED if on mouse layer
+class Layers(_Layers):
+    last_top_layer = 0
 
+    def after_hid_send(self, keyboard):
+        if keyboard.active_layers[0] != self.last_top_layer:
+            self.last_top_layer = keyboard.active_layers[0]
+            if self.last_top_layer == 6:  # mouse layer
+                led.value = True
+            else:
+                led.value = False
```

Comment:
I'd recommend wrapping `activate_layer` and `deactivate_layer` instead of constantly polling with `after_hid_send`.
example:
```python
class Layers(_Layers):
    def activate_layer(self, keyboard, layer, idx=None)
        super().activate_layer(keyboard, layer, idx)
        if keyboard.active_layers[0] == 6:
            led.value = True
```

Reply:
Thank you that makes a lot more sense! Will fix!